src/components/organisms/WorldForm/components/WorldNav/worldNavLabelMap.ts

```diff
@@ -0,0 +1,9 @@
+import { WorldNavTab } from "components/organisms/WorldForm/components/WorldNav/WorldNavTab";
+
+export const WorldNavLabelMap: Readonly<
+  Record<WorldNavTab, string>
+> = Object.freeze({
```

Comment:
Just wondering, since we already have TS's `readonly` do we need to freeze the object too?

Reply:
TS removes those on compile time. Can just remove the type, `Object.freeze()` ensures object is actually constant (and TS uses it as a hint it's dealing with `Readonly<>` type. This way I opted to use both just as a matter of course.